server/src/schema.py

```diff
@@ -26,18 +37,42 @@ class User(Base):
     team = relationship("Team", back_populates="members")
     team_name = Column(String, ForeignKey('teams.name'))
 
+    submissions = relationship("Submission", back_populates="user")
+
     def __repr__(self):
         return "<User(username='{}', name='{}', score={}, team='{}')>".format(self.username, self.name, self.score, self.team_name)
 
 
 class Challenge(Base):
-    __tablename__ = 'Challenges'
-    id = Column(Integer, autoincrement=True, primary_key=True)
+    __tablename__ = 'challenges'
+
+    identifier = Column(Integer, autoincrement=True, primary_key=True)
     value = Column(Integer)
```

Comment:
Is this the point value of the challenge? If so, that name is a little confusing, how about worth, score, or scoreWorth?